app/src/main/res/layout/activity_login.xml

```diff
@@ -36,7 +36,7 @@
         app:layout_constraintTop_toBottomOf="@+id/imageView">
 
         <com.google.android.material.textfield.TextInputEditText
-            android:id="@+id/login_username_field"
+            android:id="@+id/ComfirnPassword_field"

```

Comment:
spelling error, and use underscore between words in xml files(i.e. confirm_password_field)

Reply:
you missed this one :)

Reply:
and I think this is the login page? not sure why the id is changed to confirm_password here, login page does not have a comfirm_password field